test/Microsoft.ML.Tests/FeatureContributionTests.cs

```diff
@@ -75,15 +75,11 @@ public void TestFeatureImportance()
                 .Append(ML.Transforms.Normalize("Features"));
             var data = pipeline.Fit(srcDV).Transform(srcDV);
             var model = ML.Regression.Trainers.OrdinaryLeastSquares().Fit(data);
```

Comment:
Question: Here we cover the linear learner. FastTreePredictor also has an implementation. Should we have tests that cover each predictor implementation?

Reply:
We should definitely test each predictor's implementation. Otherwise we have no way of making sure it works.

---
In reply to: [239965259](https://github.com/dotnet/machinelearning/pull/1847#discussion_r239965259) [](ancestors = 239965259)

Reply:
Do you want to add these to this PR? Or shall we do a follow-up PR for specific tests? Either way is fine with me.

Reply:
I'll add it to this PR so that I can test that everything works for FCC.

---
In reply to: [239967406](https://github.com/dotnet/machinelearning/pull/1847#discussion_r239967406) [](ancestors = 239967406)